coop_shift/demo/shift_template_ticket.xml

```diff
@@ -10,10 +10,7 @@
     <record model="shift.template.ticket" id="template_ticket_1_standard">
         <field name="name">Standard</field>
         <field name="product_id" eval="ref('coop_shift.product_product_shift_standard')"/>
-        <field name="shift_type_id" eval="ref('coop_shift.shift_type')"/>
         <field name="shift_template_id" eval="ref('coop_shift.standard_template_1')"/>
-        <field name="start_datetime" eval="'%s-%s-%s 09:00:00' %(datetime.now().year, datetime.now().month, datetime.now().day)"/>
-        <field name="end_datetime" eval="'%s-%s-%s 12:00:00' %(datetime.now().year, datetime.now().month, datetime.now().day)"/>
```

Comment:
Don't these changes have a negative impact ?

I mean we're removing fields here, shouldn't we fix the warning by finding the right fields?

Reply:
@ivantodorovich : `start_datetime` and `end_datetime` is not exists in `shift.template.ticket` model
there is an `date_begin` but it is related fields so we don't need to add in demo data